pkg/operator/controllers/guardrails/staticresources/gk_audit_controller_deployment.yaml

```diff
@@ -100,8 +100,8 @@ spec:
             - ALL
           readOnlyRootFilesystem: true
           runAsNonRoot: true
-          seccompProfile:
-            type: RuntimeDefault
+          # seccompProfile:
```

Comment:
I do apologize. I fail to see how removing the secompProfile enables the default.
I have read the linked documentation, which explains how to add it and define. I do not see this done in this PR.

How does this work on 4.10? How does this work on 4.11?

Reply:
To clarify, I am not asking for code change, I am asking for explanation to go in as a part of the PR

Reply:
Fist of all, thanks for looking at the change, really need more eyes on guardrails related changes to prevent any potential risky code from being merged.
TBH, I'm not very sure about such security related issues, most of the changes (if not all) are due to a rising problem.
As to seccompProfile, my understanding is, for 4.10 it requires some changes described in the doc in order to make it work, otherwise the pods wont start which is the same as what I encountered recently in my tests.
and for 4.11, this has been deprecated, so with or without it wont make much difference IMO.
So far we are mostly plan to do some tests on limited clusters and it wont be enforced or largely deployed, the change should be fine for this purpose.
And I also tested another change which should be a bit securer than this, however, I'm not vert sure about that.
I was asking in OPA channel to get more insights, hopefully could address this completely.